mpi.ml

```diff
@@ -13,6 +13,7 @@
 (***********************************************************************)
 
 (* $Id$ *)
+let empty_buffer = Bytes.of_string "";;
```

Comment:
Minor cosmetic comment: the double semi-colon isn't necessary.

Reply:
In fact, there is a `Bytes.empty` (https://github.com/ocaml/ocaml/blob/9e67f4e9426ca2a8bcaeafca26dc174b88d41b1e/stdlib/bytes.mli#L78).

Reply:
Thanks! I changed it.